cmd/thanos/tools_bucket.go

```diff
@@ -616,7 +636,56 @@ func registerBucketCleanup(app extkingpin.AppClause, objStoreConfig *extflag.Pat
 	})
 }
 
-func printTable(blockMetas []*metadata.Meta, selectorLabels labels.Labels, sortBy []string) error {
+type tablePrinter func(t Table) error
```

Comment:
I'm not sure about hardcoding the output to `os.Stdout` in all the functions. What do you think about passing an `io.Writer` into any tablePrinter?

```suggestion
type tablePrinter func(w io.Writer, t Table) error
```

Reply:
Yes, even i was sceptical about using stdout directly here.